src/ConcreteOperations/isdisjoint.jl

```diff
@@ -1616,3 +1616,48 @@ end
 function is_intersection_empty(::Line2D{N}, ::EmptySet{N}) where {N<:Real}
     return true
 end
+
+# =================================================================
+# Disjointness methods for two-dimensional lines and line segments
+# =================================================================
+
+"""
+    is_intersection_empty(L1::Line2D{N}, L2::Line2D{N}, witness::Bool=false) where {N<:Real}
+
+Check whether two two-dimensional lines do not intersect.
+
+### Input
+
+- `L1` -- line
+- `L2` -- line
+
+### Output
+
+* If `witness` option is deactivated: `true` iff ``L1 ∩ L2 = ∅``
+* If `witness` option is activated:
+  * `(true, [])` iff ``L1 ∩ L2 = ∅``
+  * `(false, v)` iff ``L1 ∩ L2 ≠ ∅`` and ``v ∈ L1 ∩ L2``
+"""
+function is_intersection_empty(L1::Line2D{N}, L2::Line2D{N}, witness::Bool=false) where {N<:Real}
+    disjoint = _isdisjoint(L1, L2)
+    if !witness
+        return disjoint
+    else
+        if disjoint
+            return (true, [])
+        else
+            return (false, an_element(intersection(L1, L2)))
```

Comment:
(`intersection(L1, L2)` recomputes many things, including the determinant. We could have a helper method that gives the intersection point. But we can do that another time.)

Reply:
true. i've opened a new issue